src/config.ts

```diff
@@ -205,6 +206,10 @@ export class Configuration {
         process.env[k] = current.join(",")
       }
 
+      if (typeof current === "boolean") {
+        process.env[k] = String(current)
+      }
+
```

Comment:
The check below this one would set it again if `true` (which wouldn't be a problem), or a non-boolean (which could be a problem), no?
We could make it an `else if (current)` statement part of the `if (typeof current === "boolean")` one.